app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -103,12 +103,23 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       function setHiddenValues () {
         that.$somethingIsWrongForm.append('<input type="hidden" name="javascript_enabled" value="true"/>')
         that.$somethingIsWrongForm.append($('<input type="hidden" name="referrer">').val(document.referrer || 'unknown'))
+        that.$somethingIsWrongForm.data('invalidInfoError', [
+          '<h2>',
+          '  Sorry, we’re unable to send your message as you haven’t given us any information.',
+          '</h2>',
+          '<p>Please tell us what you were doing or what went wrong</p>'
+        ].join(''))
       }
 
       function setHiddenValuesNotUsefulForm (gaClientId) {
         var currentPathName = window.location.pathname.replace(/[^\s=?&]+(?:@|%40)[^\s=?&]+/, '[email]')
         var finalPathName = encodeURI(currentPathName)
-
+        that.$surveyForm.data('invalidInfoError', [
```

Comment:
Appreciate that we want to fix this quickly - how much work would it be to not use jQuery for the changes in this pull request?

Reply:
Short answer yeah it is possible to not use jQuery but it [ends up more verbose](https://github.com/alphagov/govuk_publishing_components/pull/1684/commits/7b93e353e7106d82575b1644add14643197c9266) because everything in this module is reliant on jQuery (and now I'm itching to convert the whole thing to vanilla js but I can't do it in this PR because it's not within the scope of this work 😬)